alert.go

```diff
@@ -910,6 +916,31 @@ func (a *AlertNode) handlePost(post *pipeline.PostHandler, ad *AlertData) {
 	return
 }
 
+func (a *AlertNode) handleTcp(tcp *pipeline.TcpHandler, ad *AlertData) {
+	bodyBuffer := a.bufPool.Get().(*bytes.Buffer)
+	defer func() {
+		bodyBuffer.Reset()
+		a.bufPool.Put(bodyBuffer)
+	}()
+
+	err := json.NewEncoder(bodyBuffer).Encode(ad)
+	if err != nil {
+		a.logger.Println("E! failed to marshal alert data json", err)
+		return
+	}
+
+	conn, err := net.Dial("tcp", tcp.Address)
```

Comment:
I had considered that, but it seemed like it would be a pretty large undertaking. I'm assuming it would require some connection pooling per handler, with some mechanisms for checking the health of the connections (and maybe closing connections after a longer period of inactivity). Not sure if that's way off base or not, though. 

I'd be up for discussing it more in depth offline if you have some time.